src/index.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom';
+
+function Counter({ count, onClick }) {
+    return (
+        <button type="button" onClick={() => onClick(1)}>
```

Comment:
```suggestion
        <button 
          type="button"
          onClick={() => onClick(1)}
         >
```

비교했을 떄 어떤게 더 읽기 편한가요?
저는 속성이 2개 이상일 떄는 줄바꿈을 해주는 편인데 비교해보시고 자기만의 코드 정리 기준을 만들어 보셔도 좋아요!

Reply:
이렇게 한 번 해보겠습니다~!